tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -310,6 +310,9 @@ generate_qemu_options() {
 	qemu_options+=(size:--disable-qom-cast-debug)
 	qemu_options+=(size:--disable-tcmalloc)
 
+	# Disable libudev for static build on ppc64le
+	[ "$arch" == "ppc64le" ] && qemu_options+=(size:--disable-libudev)
```

Comment:
is this needed only for static build? maybe you want to check if  `"${static}" == "true"` first ... ?

Reply:
yes, that would be better. Thanks @devimc, updated.